pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -9,8 +9,6 @@ CREATE SCHEMA IF NOT EXISTS SCHEMA_METRIC;
 CREATE SCHEMA IF NOT EXISTS SCHEMA_DATA;
 CREATE SCHEMA IF NOT EXISTS SCHEMA_INFO;
 
-CREATE EXTENSION IF NOT EXISTS timescaledb WITH SCHEMA public;
```

Comment:
should this be moved out of here? Why was this moved?

Reply:
if we leave it hear the migrate fails with
```SQL
ERROR: parameter "extwlist.extensions" must be a list of extension names
```
I suspect the migrate library is doing something non-standard with connections, which is causing the guc to not load correctly.